app/components/dashboard/patient_breakdown_component.html.erb

```diff
@@ -26,21 +30,12 @@
       <%= number_with_delimiter(data[:ltfu_patients]) %>
     </p>
   </div>
-  <div class="d-flex ai-center jc-between">
-    <p class="mb-8px">
-      Dead patients
-    </p>
-    <p class="mb-8px" id="dead-patients">
-      <%= number_with_delimiter(data[:dead_patients]) %>
-    </p>
-  </div>
 </div>
 <div class="d-flex ai-center jc-between">
-  <p class="mb-8px fw-bold">
```

Comment:
[question]: What the reason behind removing the cumulative registrations from the patient coverage graph?

Reply:
Because we're going to show this at the to with the new graph